src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/basic.html.twig

```diff
@@ -28,6 +28,29 @@
 <div role="tabpanel" class="form-contenttab tab-pane container-fluid active" id="basic-tab">
   <div class="row">
     <div class="col-md-9 left-column">
+      <div 
+        id="product-images-container"
+        data-translations="{{
+          {
+            'window.selectAll': 'Select all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.unselectAll': 'Unselect all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.selectedFiles': 'selected file(s)'|trans({}, 'Admin.Catalog.Feature'),
+            'window.useAsCover': 'Use as cover image'|trans({}, 'Admin.Catalog.Feature'),
+            'window.saveImage': 'Save image settings'|trans({}, 'Admin.Catalog.Feature'),
```

Comment:
This one exists in `Admin.Actions` but I don't understand why, it's not a generic wording it's only related to the Product image edition So your choice of `Admin.Catalog.Feature` makes more sense

@Julievrz what do you think?

Reply:
If it is only used for the Product image edition then you are right to say it should not be in Admin.Actions. However, I feel like it's generic enough to be in Admin.Actions. I could imagine this string to be reused later on another page.

Reply:
Only the product has advanced Image settings for now (with legend, cover, ...) I don't think other entities will have advanced image settings in the near future
But we can keep the `Admin.Actions` anyway since it already exists I guess